package/lib-tensorflow_cc-shared-1.4.0/scripts.android/install.bat

```diff
@@ -0,0 +1,41 @@
+@echo off
+
+rem
+rem Installation script.
+rem
+rem See CK LICENSE.txt for licensing details.
+rem See CK COPYRIGHT.txt for copyright details.
+rem
+rem Developer(s):
+rem - Grigori Fursin, grigori@dividiti.com, 2016-2017
+rem - Anton Lokhmotov, anton@dividiti.com, 2017
+rem
+
+if not "%CK_HAS_OPENMP%" == "0" (
+  set CK_REF_LIBRARIES=%CK_LINKER_FLAG_OPENMP%
+)
+
+xcopy /E %ORIGINAL_PACKAGE_DIR%\*.patch %INSTALL_DIR%\obj
+
+rem To avoid problem linking OpenCL lib ...
+
+set CMAKE_C_FLAGS=%CMAKE_C_FLAGS% -L%CK_ENV_LIB_OPENCL_LIB%
+set CMAKE_CXX_FLAGS=%CMAKE_CXX_FLAGS% -L%CK_ENV_LIB_OPENCL_LIB%
+
+set JOB_COUNT=%CK_HOST_CPU_NUMBER_OF_PROCESSORS%
+
+set CK_CMAKE_EXTRA=%CK_CMAKE_EXTRA% ^
+ -DOPENCL_ROOT:PATH="%CK_ENV_LIB_OPENCL%" ^
+ -DOPENCL_LIBRARIES:FILEPATH="%CK_ENV_LIB_OPENCL_DYNAMIC_NAME%" ^
+ -DOPENCL_INCLUDE_DIRS:PATH="%CK_ENV_LIB_OPENCL_INCLUDE%" ^
+ -DCMAKE_EXE_LINKER_FLAGS=-L"%CK_ENV_LIB_OPENCL_LIB%" ^
+ -DTUNERS=ON ^
+ -DCLTUNE_ROOT:PATH="%CK_ENV_TOOL_CLTUNE%" ^
```

Comment:
This is odd. Why are OpenCL and CLTune needed?

Reply:
Not sure. It is the same as in package for TF 1.3. Should I check building for android?

Reply:
Do you have a recent Android device?